### DISCUSSION TITLE
last 3 update fail to load with error

### DISCUSSION THREAD

**Discussion by stableartai** [zanllp/infinite-image-browsing#663] [Q&A]
After attempt to last three update still get a load error failure: timestamp out of range for platform time_t

any ideas on why it keep getting error. have to downrev so continue to use older version.

**Comment by zanllp** [comment]
Have you tried this branch yet?<img width="857" alt="image" src="https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/9c2399b3-364b-4fc2-9b2a-e862911438e1">

**Comment by zanllp** [comment]
Just upgrade to the latest version, I've merged the previous branch into it.

**Comment by stableartai** [comment]
would if knew where such option of enhance-compatibility was located.

**Comment by stableartai** [comment]
Ok looks like we remove and did the url install and seem to be loading the one from 6-22 and no error, double checking
